icons/shredder.json

```
@@ -0,0 +1,33 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Alirashidy",
```

Those are the people who created the `file` icon, since I based this icon on `file`, we should include them.
```suggestion
    "Alirashidy",
    "colebemis",
    "ericfennis",
    "danielbayley",
```

---

We should also include [karsa-mistmere](https://github.com/karsa-mistmere), for having the suggestion with the shorter lines.

---

Yay! Thank a lot, I think I did it right. I also added @karsa-mistmere, thanks for the suggestion!